docs/workflows.rst

```diff
@@ -256,34 +285,16 @@ Slice time correction
                   'SliceTiming': [0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9]},
         )
 
-If the ``SliceTiming`` field is present in the input dataset metadata, this
-workflow will be included to perform slice time correction prior to head motion
-estimation.
-Slice time correction is performed using AFNI 3dTShift.
+If the ``SliceTiming`` field is available within the input dataset metadata,
+this workflow performs slice time correction prior to other signal resampling
+processes.
+Slice time correction is performed using AFNI ``3dTShift``.
 All slices are realigned in time to the middle of each TR.
 
-Slice time correction can be disabled with ``--ignore slicetiming`` command
-line argument.
-If a BOLD series has fewer than 5 usable (steady-state) volumes, slice time
-correction will be disabled.
-
-.. _bold_hmc:
-
-Head-motion estimation
-~~~~~~~~~~~~~~~~~~~~~~
-:mod:`fmriprep.workflows.bold.hmc.init_bold_hmc_wf`
-
-.. workflow::
-    :graph2use: colored
-    :simple_form: yes
-
-    from fmriprep.workflows.bold import init_bold_hmc_wf
-    wf = init_bold_hmc_wf(
-        mem_gb=1,
-        omp_nthreads=1)
-
-FSL MCFLIRT is used to estimate motion transformations using an automatically
-estimated reference scan (see :ref:`bold_ref`).
+Slice time correction can be disabled with the ``--ignore slicetiming``
+command line argument.
+If a :abbr:`BOLD (blood-oxygen level-dependent)` series has fewer than
+5 usable (steady-state) volumes, slice time correction will be disabled.
```

Comment:
Perhaps "will be automatically disabled for that run." to make clear that it's on a per-run basis.